tools/sigclient/pkg/utils/reader.go

```diff
@@ -36,12 +36,13 @@ import (
 )
 
 var json = jsoniter.ConfigFastest
+var uuidList []string
+var mu sync.Mutex
```

Comment:
nit: especially since this is a global variable, use a more descriptive name than `mu`, or at least comment what this mutex is used for